plugins/poll/assets/javascripts/components/poll-voters.js.es6

```diff
@@ -1,13 +1,49 @@
+import { on } from 'ember-addons/ember-computed-decorators';
+
 export default Ember.Component.extend({
   layoutName: "components/poll-voters",
   tagName: 'ul',
   classNames: ["poll-voters-list"],
   isExpanded: false,
-  numOfVotersToShow: 20,
+  numOfVotersToShow: 0,
+  offset: 0,
+  loading: false,
+  pollsVoters: [],
+
+  _fetchUsers() {
+    this.set("loading", true);
+
+    Discourse.ajax("/polls/voters.json", {
+      type: "get",
+      data: { user_ids: this.get("voterIds") }
+    }).then(result => {
+      if (this.isDestroyed) return;
+      this.set("pollsVoters", this.get("pollsVoters").concat(result.users));
+      this.incrementProperty("offset");
+      this.set("loading", false);
+    }).catch((error) => {
+      console.log(error);
+      bootbox.alert(I18n.t('poll.error_while_fetching_voters'));
+    });
+  },
+
+  _getIds(ids) {
+    const numOfVotersToShow = this.get("numOfVotersToShow");
+    const offset = this.get("offset");
+    return ids.slice(numOfVotersToShow * offset, numOfVotersToShow * (offset + 1));
+  },
+
+  @on("didInsertElement")
```

Comment:
I used to prefer `on('didInsertElement')` because it means you don't have to call `this._super()` but after talking with ember core I'm told it can't be optimized as well, so I've gone back to using `didInsertElement() { this._super(); .... `